hack/db_copy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#TO_PSW=postgres
+to_host=rhdb.cx6ygmy4go45.eu-north-1.rds.amazonaws.com
```

Comment:
Why is this hard-coded? This should be an input argument, no? Same comments about the other `{from,to}_*` parameters. Or if it is provided as an example, maybe you could rename it accordingly, like: `db_copy.example.sh`.

Reply:
well, it is stated that one have to create copy of this file and provide values here.
I am not sure arguments are more convenient than modifying fields here especially that many of the fields most probably will be unchanged.
So, I would "recommend" password only  for argument, for security reason.
WDYT?

Reply:
Alright. Fine then if it is stated that a copy of the file should be created.